src/crimson/os/seastore/async_cleaner.h

```diff
@@ -385,9 +390,10 @@ class ExtentCallbackInterface {
   auto do_with_transaction_intr(
       Transaction::src_t src,
       const char* name,
+      cache_hint_t fadv,
       Func &&f) {
     return seastar::do_with(
-      create_transaction(src, name, IsWeak),
+      create_transaction(src, name, fadv, IsWeak),
```

Comment:
if convenient, rename fadv* inside seastore (async_cleaner, cache, transaction_manager) to cache_hint.